tests/models/test_evaluation.py

```diff
@@ -136,6 +138,57 @@ def breast_cancer_dataset():
     return ds
 
 
+def get_pipeline_model_dataset():
+    """
+    The dataset tweaks the IRIS dataset by changing its features into categorical features.
+    The dataset is prepared for a pipeline model
+    (contains an indexer on feature "f1" and a classifier).
+    """
+    X, y = get_iris()
+
+    def convert_num_to_label(x):
+        return f"v_{round(x)}"
+
+    data = pd.DataFrame(
+        {
+            "f1": np.array(map(convert_num_to_label, X[:, 0])),
+            "f2": np.array(map(convert_num_to_label, X[:, 1])),
+            "y": y,
+        }
+    )
+    return data, "y"
+
+
+@pytest.fixture(scope="module")
+def pipeline_model_dataset():
+    data, target_col = get_pipeline_model_dataset()
+    constructor_args = {"data": data[0::3], "targets": target_col, "name": "pipeline_model_dataset"}
+    ds = EvaluationDataset(**constructor_args)
+    ds._constructor_args = constructor_args
+    return ds
```

Comment:
Can we remove this fixture and directly use `get_pipeline_model_dataset` in `test_pipeline_model_kernel_explainer_on_categorical_features`?